api/src/configure.test.ts

```diff
@@ -196,10 +188,12 @@ export function builderTestEnvironment(prefix: string) {
     //   workspace_id: workspace1.id,
     // })
     const userPassword = 'locokit'
-    const [localStrategy] = app.service('authentication').getStrategies('local') as LocalStrategy[]
+    const [localStrategy] = app
+      .service(SERVICES.AUTH_AUTHENTICATION)
+      .getStrategies('local') as LocalStrategy[]
     const passwordHashed = await localStrategy.hashPassword(userPassword, {})
     // @ts-expect-error this should work as expected but don't respect the schemas
-    user1 = await app.services.user._create(
+    user1 = await app.service(SERVICES.CORE_USER)._create(
```

Comment:
None of these users have a firstName, lastName, we need this information to test filters on them

Reply:
Do you mean you want firstName & lastName to be mandatory fields ?

Reply:
Updated in #232